src/Components/Form/FormFields/RangeFormField.tsx

```diff
@@ -18,6 +18,7 @@ type BaseProps = FormFieldBaseProps<number> & {
   valueDescriptions?: ValueDescription[];
   hideInput?: boolean;
   hideUnitInLabel?: boolean;
+  allowIntegersOnly?: boolean;
```

Comment:
just a thought, we could automatically infer whether to allow integers only or not by checking against `step` value right?

Reply:
🤔  yea that will work for almost all fields (except insulin intake dosage, so we can either just add step of 0.1 there, so disallow decimal entries for the field).

Reply:
@rithviknishad what do you suggest? ^

Reply:
we can set the step `0.1` for that field.